app/charts/combo/axis-height-linear-dual.tsx

```diff
@@ -65,6 +65,9 @@ export const AxisHeightLinearDual = (props: AxisHeightLinearDualProps) => {
     TITLE_HPADDING * (isOverlapping ? Math.floor(overlapAmount) : 2);
 
   const titleWidth = isOverlapping ? axisTitleWidth / overlapAmount : "auto";
+  const axisLabelHeight = getTextHeight(axisTitle, {
+    fontSize: axisLabelFontSize,
+  });
```

Comment:
@noahonyejese I believe here we need to pass `width: chartWidth / 2`, otherwise the height is computed based on full chart width, which is not correct?

Reply:
I think this comment should also be applied to other places where we have two titles that can overlap (combo line column chart) 👀 

Let me know if I missed something!